app/models/case.rb

```diff
@@ -12,6 +12,10 @@ def complete
     end
   end
 
+  validates :date_case_concluded, presence: true, if: -> { has_case_concluded == 'yes' }
+  validates :date_client_remanded, presence: true, if: -> { is_client_remanded == 'yes' }
+  validates :preorder_work_date, :preorder_work_details, presence: true, if: -> { is_preorder_work_claimed == 'yes' }
+
```

Comment:
Not sure what best practice is - whether to use the constant for the yes value or ok to just compare to 'yes'?

Reply:
We need to declare global constants somewhere. So i added new file. Thanks for pointing it out. 👏